collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/utils/AdTagFactory.kt

```diff
@@ -0,0 +1,17 @@
+package com.bitmovin.analytics.bitmovin.player.utils
+
+import com.bitmovin.analytics.ads.AdTagType
+import com.bitmovin.player.model.advertising.AdTag
+
+class AdTagFactory {
+
+    fun FromPlayerAdTag(adTag: AdTag): AdTagType? {
```

Comment:
Do we really want to make this nullable?

Reply:
I added the AdTagType UNKNOWN and return it if incoming adTagType is not convertable